### ISSUE TITLE
Text instead of colored dots

### ISSUE THREAD

**Issue by MeDustyy** [AlexisL61/JackboxUtility#167]
I think these dots might confuse some users of the app (users may think it is ON/OFF, not YES/NO). Therefore, I suggest replacing them with a text version that will be aligned on the right side.
![Знімок екрана 2023-06-23 1033072](https://github.com/AlexisL61/JackboxUtility/assets/58863601/29618719-1707-4281-a3f0-69fe8785d82b)

**Comment by AlexisL61** [comment]
@AkiraArtuhaxis what do you think about it ?

**Comment by AkiraArtuhaxis** [comment]
I honestly don't think those dots cause confusion to the users. Never in the app can you click those dots to change them (in the patcher, in the translation tab and here in the game info tab). They always signify the state of that setting.
Despite this looking like an alright change, the problem is that "MILDLY PLAYABLE" and "FULL MODERATION" probably won't fit in there. Plus, this can _also_ confuse users regardless (they might think you can click those words to change them).
In conclusion, I think the dots are fine as they are.

**Comment by AkiraArtuhaxis** [comment]
The problem with the text is that it also causes a small contradiction that you can even see in the image. Games that don't require moderation are stated as "Moderation: YES".
